### PR TITLE
Add default device type

### PR DESCRIPTION
**Pull request by cehali**
# Summary
Add default device type

### CODE REVIEW

**Comment by cehali** [comment]
image
image

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ interface LogoProps {
 export const Logo = ({logo, applicationName, deviceType}: LogoProps) => {
   const src = isLogoEmpty(logo) ? DEFAULT_LOGO : logo;
   return (
-    <div className={`connected-devices-img-wrapper ${deviceType}`}>
+    <div className={`connected-devices-img-wrapper ${deviceType !== 'unknown' ? deviceType : 'default'}`}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: Why introduce unnecessary complexity? Change CSS instead:
`.connected-devices-img-wrapper.default::before` 
->   
`.connected-devices-img-wrapper.unknown::before`

File: universal-login-wallet/src/ui/assets/icons/U.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...52.348l3.523-1.21 5.867-2.19 2.84.748 3.182-1.14 3.574-3.047 2.93-1.483 1.358-3.984.484-3.322.732-2.387v-3.942l.445-3.625v-2.897l-.356 2.529a22.365 22.365 0 01-1.071 4.368l-.321.905a6.92 6.92 0 00-.396 2.525 3.221 3.221 0 01-.835 2.262l-.56.617a11.638 11.638 0 01-3.334 2.547L20.895 45.5l-.282 1.388-.573 1.425-.277.425-.084.11c-.12.16-.246.314-.377.465a5.71 5.71 0 01-.414.43l-.022.02c-.16.15-.329.29-.506.418l-.152.11-.632.423-3.208 1.634z" fill="url(#i)" opacity=".498" transform="rotate(117 26.792 37.98)"/><path d="M.413.768h6.42c7.18 0 13 5.82 13 13v10.249a6.295 6.295 0 006.295 6.295c4.087 0 8.117-.957 11.768-2.793l.284-.143a25.022 25.022 0 009.492-8.35l.023-.033a22.98 22.98 0 003.92-12.016l.229-6.209v22.9c0 14.203-11.514 25.716-25.716 25.716C11.926 49.384.413 37.871.413 23.67V.769z" fill="url(#j)" transform="translate(-.177 -.147)"/><path d="M.413.768h6.42c7.18 0 13 5.82 13 13V43.09a6.295 6.295 0 006.295 6.294C11.926 49.384.413 37.871.413 23.67V.769z" fill="url(#k)" transform="translate(-.177 -.147)"/><path d="M.413.768h6.42c7.18 0 13 5.82 13 13V43.09a6.295 6.295 0 006.295 6.294C11.926 49.384.413 37.871.413 23.67V.769z" fill="url(#l)" transform="translate(-.177 -.147)"/></g></svg>
```
Comment (marekkirejczyk) [inline universal-login-wallet/src/ui/assets/icons/U.svg]: It it browser icon? It should be a browser icon. Name suggest UL icon.

**Comment by cehali** [comment]
image
image

File: universal-login-wallet/src/ui/assets/icons/U.svg (hunk 2 above)
Comment (marekkirejczyk) [reply universal-login-wallet/src/ui/assets/icons/U.svg]: This file is now unused, can be removed.

**Pull request merged**